templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete Manufacturer</h1>
+  
+  <p>Are you sure you want to delete this Manufacturer: {{ manufacturer.name }}?</p>
+  <p><i style="color: red; background: yellow">Attention! All cars of this manufacturer will also be deleted!!!</i></p>
+
+  <form action="" method="post">    
+    {% csrf_token %}
+    <input class="btn btn-danger" type="submit" value="Yes, delete">
+    <a class="btn btn-success" href="{% url 'taxi:manufacturer-list' %}">No, cancel</a>

```

Comment:
According to the checklist, the 'Cancel' button should lead to the previous page the user was on, not a fixed URL. Consider using JavaScript to navigate back to the previous page, such as using `window.history.back()`.